pkg/alert/ctl_alert.go

```diff
@@ -147,6 +147,33 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.DesiredState, "alert-desired-state", ctl.DesiredState, "State of the Alert")
 }
 
+var nSpecFlagCmd cobra.Command // command for getting flags for this spec
```

Comment:
Why declare as a global variable?  This seems to be used temporarily vs needing to keep the value.  Just have temp variables where this is used assigned with :=.